apps/articles/migrations/0009_load_creators.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.2.13 on 2022-06-07 07:31
+
+from django.db import migrations
+from django.db.models import IntegerField
+from django.contrib.admin.models import LogEntry, ADDITION
+from django.contrib.contenttypes.models import ContentType
+from django.db.models.functions import Cast
+
+
+def load_creators(apps, schema_editor):
+    def load_using_model_name(model_name):
+        Model = apps.get_model("articles", model_name)
+        contentType = ContentType.objects.get_for_model(Model)
+        model_logs = LogEntry.objects\
+            .annotate(object_id_as_int=Cast('object_id', IntegerField()))\
+            .filter(object_id_as_int__in=Model.objects.values_list('id'))\
+            .filter(content_type=contentType)\
+            .filter(action_flag=ADDITION)\
+            .values("object_id_as_int", "user_id")
+        for log in model_logs:
+            model = Model.objects.filter(id=log["object_id_as_int"])
+            if model.first().creator is None:
+                model.update(creator=log["user_id"])
+
+    load_using_model_name("Project")
```

Comment:
Можно было передавать модель напрямую, минус одна строка в функции, но и так тоже норм
```suggestion
    load_using_model_name(Project)
```

Reply:
![image](https://user-images.githubusercontent.com/82268688/174526747-c37316b0-d751-4dda-ba06-58e4ec1064ac.png)